types/jsreport-phantom-pdf/index.d.ts

```diff
@@ -0,0 +1,51 @@
+/// <reference types='jsreport-core' />
+
+declare namespace JsReport {
+  const enum RecipeType {
+    PhantomPdf = 'phantom-pdf'
+  }
+
+  interface Margin {
+    left: number | string;
+    right: number | string;
+    top: number | string;
+    bottom: number | string;
+  }
+
+  interface Phantom {
+    margin: string | Margin;
+    header: string;
+    footer: string;
+    width: string;
+    height: string;
+    headerHeight: string;
+    footerHeight: string;
+    format: string;
+    orientation: 'portrait' | 'landscape';
+    blockJavaScript: 'true' | any;
+    resourceTimeout: number;
+    waitForJS: 'true' | 'false';
+    fitToPage: 'true' | 'false';
+    customPhantomJS: 'true' | 'false';
+    phantomjsVersion: string;
+  }
+
+  interface Template {
+    phantom?: Partial<Phantom>;
+  }
+}
+
+declare namespace JsReportPhantomPdf {
+  interface Options {
+    allowLocalFilesAccess: boolean;
+    appDirectory: string;
+    defaultPhantomjsVersion: string;
+    strategy: string;
+  }
```

Comment:
Please include `timeout: number`. The `appDirectory` on the other hand should not be there. Could the strategy be also an enum? 
```js
const enum PhantomStrategy {
  dedicatedProcess = 'dedicated-process',
  phantomServer = 'phantom-server'
}
```

Reply:
OK, i will make another pr later. actually, I don't know much about this project, I declared the types just by searching the source code.

Reply:
Nothing to worry about. I am not an expert in typescript. So great that we can finish this together. :) Thank you for your time.

Reply:
It's all right. this project is so great that I just want to help more people.